ui/packages/ui/src/app/pages/createConnector/CreateConnectorPage.tsx

```diff
@@ -207,10 +219,13 @@ export const CreateConnectorPage: React.FunctionComponent = () => {
   ): void => {
     setBasicPropValues(basicPropertyValues);
     setAdvancedPropValues(advancePropertyValues);
+    // Don't include connector name for validation
+    const basicForValidation = new Map<string,string>(basicPropertyValues);
+    basicForValidation.delete(PropertyName.CONNECTOR_NAME);
```

Comment:
why do we need to delete the CONNECTOR_NAME before calling validation?

Reply:
as it's currently already getting passed to validate in the filter step.

Reply:
I removed it because it is an extra value that is not needed for backend validation.  However, it appears that the backend just ignores it - so it should not be a problem to leave it for now...